pmm-admin_test.go

```diff
@@ -1389,6 +1394,100 @@ func testAddMySQL(t *testing.T, data pmmAdminData) {
 	assertRegexpLines(t, expected, string(output))
 }
 
+func testAddMySQLWithCreateUser(t *testing.T, data pmmAdminData) {
```

Comment:
`testAddMySQLWithCreateUser` is unused

Reply:
I know. Let's leave it, it's commented why it's not used.